core/lib/via_btc_client/examples/bootstrap.rs

```
@@ -47,141 +80,250 @@ async fn main() -> Result<()> {
-    let bridge_musig2_address = "bcrt1p3s7m76wp5seprjy4gdxuxrr8pjgd47q5s8lu9vefxmp0my2p4t9qh6s8kq"
-        .parse::<BitcoinAddress<NetworkUnchecked>>()?;
-
-    let mut verifier_inscribers: Vec<Inscriber> = vec![
-        create_inscriber(
-            &verifier_1_private_key,
-            &rpc_url,
-            &rpc_username,
-            &rpc_password,
-            network,
-        )
-        .await?,
-        create_inscriber(
-            &verifier_2_private_key,
-            &rpc_url,
-            &rpc_username,
-            &rpc_password,
-            network,
-        )
-        .await?,
-        // create_inscriber(
-        //     &verifier_3_private_key,
-        //     &rpc_url,
-        //     &rpc_username,
-        //     &rpc_password,
-        //     network,
-        // )
-        // .await?,
-    ];
+    let mut inscriber = create_inscriber(
+        &private_key,
+        &rpc_url,
+        &rpc_username,
+        &rpc_password,
+        network,
+    )
+    .await?;
+
+    match inscription {
+        InscriptionType::SystemBootstrapping => {
+            let system_tx_id = bootstrap_inscription(&args, &mut inscriber, network).await?;
+            let propose_sequencer_tx_id = propose_sequencer(&args, &mut inscriber, network).await?;
+            let mut data = HashMap::new();
+            data.insert("system_tx_id".to_string(), system_tx_id.to_string());
+            data.insert(
+                "propose_sequencer_tx_id".to_string(),
+                propose_sequencer_tx_id.to_string(),
+            );
+            data.insert(
+                "tx_type".to_string(),
+                InscriptionType::SystemBootstrapping.to_string(),
+            );
+            let json_data = to_string_pretty(&data).expect("Failed to serialize");
+
+            let dir = format!("etc/env/via/genesis/{}", network,);
+
+            if network == BitcoinNetwork::Regtest {
+                remove_dir_all(dir.clone())?;
```

This function fails if the path does not exist, we should ignore the error

---

Fixed [commit](https://github.com/vianetwork/via-core/pull/160/commits/a7d4f3127a30a441c5631cfe6f5d46287ee99756)